drp_python/translation_layer/machines_translation.py

```diff
@@ -69,6 +69,16 @@ def delete_machine(self, machine_uuid):
         logger.warning('Deleted ' + machine_uuid)
         return
 
+    def add_machine_params(self, params_config_model, machine_uuid):
+        logger.warning('add params to machine')
+        value = params_config_model.value
+        param = params_config_model.name
+        temp = self.session.post('machines/' + machine_uuid +
+                                 '/params/' + param, value)
+        machine_model = self.get_machine(machine_uuid)
+        logger.warning('Updated ' + machine_uuid)
```

Comment:
In general, warning logger level should be something that needs attention. Should not this be info level instead. There are other similar logging messages in this category as well.

Reply:
woops left in for testing.  good catch